pkg/util/errors.go

```diff
@@ -1,26 +0,0 @@
-/*
-Copyright 2022 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package util
-
-import (
-	"net/http"
-	"strings"
-)
-
-func IsNotFoundError(err error) bool {
```

Comment:
I think we should keep this around in the old version.

Reply:
I think we don't have to be as strict as in core CAPI. In core CAPI we have to assume that every exported func can be used by someone importing CAPI as a library.

I wouldn't be as strict in CAPV